.github/workflows/backend.yml

```diff
@@ -0,0 +1,103 @@
+name: Backend
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release**'
+      - 'feature**'
+      - 'fix**'
+      - 'ci**'
```

Comment:
These should be ran on every branch, always... If I name my branch `refactor-something` then they wouldn't run with these patterns

Reply:
Actually this is a list of conditions _only_ for direct push or merge.

For pull requests, **regardless** of the name of branch, the workflow is always triggered (due to `pull_request:` key earlier, which doesn't specify the branch values).